js/main.js

```diff
@@ -4,19 +4,121 @@ var fullName = document.querySelector('#fullName');
 var $location = document.querySelector('#location');
 var bio = document.querySelector('#bio');
 var image = document.querySelector('.image');
-var button = document.querySelector('button');
+var container = document.querySelector('.container');
+var form = document.querySelector('form');
 
 avatar.addEventListener('input', function () {
   image.setAttribute('src', avatar.value);
 });
 
-button.addEventListener('click', function () {
+form.addEventListener('submit', function (event) {
+  event.preventDefault();
+
   data.profile.avatarUrl = avatar.value;
   data.profile.username = username.value;
   data.profile.fullName = fullName.value;
   data.profile.location = $location.value;
   data.profile.bio = bio.value;
   var dataVar = JSON.stringify(data);
   localStorage.setItem('data-object', dataVar);
-  document.reset();
+  viewSwapping('profile');
+  form.reset();
+});
+
+document.addEventListener('DOMContentLoaded', function () {
+  var data = localStorage.getItem('data-object');
+  data = JSON.parse(data);
+  if (data.profile.username !== '') {
```

Comment:
dont use local storage to get your data, the data object already contains that data